### PR TITLE
Add decimal ~> 2.0 to dependencies

### PR DESCRIPTION
**Pull request by andreyuhai**
Add `decimal ~> 2.0` to dependencies, not forcing current users of the package to update.

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -37,7 +37,7 @@ defmodule AbsintheHelpers.MixProject do
       {:ex_doc, "~> 0.34", only: :dev, runtime: false},
       {:absinthe, "~> 1.0"},
       {:mimic, "~> 1.10", only: :test},
-      {:decimal, "~> 1.9"}
+      {:decimal, "~> 2.0"}
```
Comment (mpmiszczyk) [inline mix.exs:40]:
```suggestion
      {:decimal, "~> 1.9 or ~> 2.0"}
```

Comment (mpmiszczyk) [reply mix.exs:40]: Let's not force update for user of this library

**Review by mpmiszczyk — CHANGES_REQUESTED** [review]

File: mix.exs (hunk 1 above)
Comment (andreyuhai) [reply mix.exs]: Had not thought about that. 🙇

**Review by mpmiszczyk — COMMENTED** [review]
Looks like we are not using any of the breaking API

**Review by mpmiszczyk — APPROVED** [review]

**Pull request merged**